docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -1,29 +1,22 @@
 ---
-sidebar_label: '🌄 Create LSP7 Token'
+sidebar_label: '🌄 Create an LSP7 Token'
 sidebar_position: 2
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Create LSP7 Token
+# Create an LSP7 Token
 
-:::caution Disclaimer
+:::tip
 
-This guide might contain outdated information and will be updated soon.
+Please check the [Getting Started](./getting-started.md) for the project setup.
 
 :::
 
 ## Create a custom LSP7 Token contract
```

Comment:
```suggestion
## Create a custom LSP7 Token
```

Try to keep the headings as small but concise as possible. Otherwise, they might not fit on the side-bar.